ci/linux-debian.Dockerfile

```diff
@@ -11,21 +11,25 @@ RUN dpkg --add-architecture i386 && \
 RUN apt-get update && apt-get install --no-install-recommends -y \
         git ca-certificates \
         make automake libtool pkg-config dpkg-dev valgrind qemu-user \
-        gcc clang llvm libc6-dbg \
+        gcc clang llvm libclang-rt-dev libc6-dbg \
         g++ \
-        gcc-i686-linux-gnu libc6-dev-i386-cross libc6-dbg:i386 libubsan1:i386 libasan6:i386 \
+        gcc-i686-linux-gnu libc6-dev-i386-cross libc6-dbg:i386 libubsan1:i386 libasan8:i386 \
         gcc-s390x-linux-gnu libc6-dev-s390x-cross libc6-dbg:s390x \
         gcc-arm-linux-gnueabihf libc6-dev-armhf-cross libc6-dbg:armhf \
         gcc-aarch64-linux-gnu libc6-dev-arm64-cross libc6-dbg:arm64 \
         gcc-powerpc64le-linux-gnu libc6-dev-ppc64el-cross libc6-dbg:ppc64el \
-        gcc-mingw-w64-x86-64-win32 wine64 wine \
+        gcc-mingw-w64-x86-64-win32 wine64 \
```

Comment:
When I added this, the duplication was intentional. If every step in the dockerfile lists its dependencies, it's easier to remove steps. For example, if we ever want to get rid of `msvc-wine`, we can just drop that `RUN` step and the rest of the image is unaffected.

Reply:
I see. It seems the `wine` package is not necessary for cross-compiling tasks anyway. But I need to test this assumption once more.

Reply:
It installs the `wine` symlink. I think that's convenient.

Reply:
True. Reverted back.